### ISSUE TITLE
2.25 update breaks formatting of for loop with parentheses around start value

### ISSUE THREAD

**Issue by joshtynjala** [vshaxe/vshaxe#545]
Original:

```haxe
for (i in (startLine + 1)...endLine)
```

Wrong formatting:

```haxe
for (i in(startLine + 1)...endLine)
```

**Comment by AlexHaxe** [comment]
seems like that's broken in latest formatter. apparently none of the 742 testcases caught it. I'll add one more then. :)

**Comment by AlexHaxe** [comment]
fixed in v2.26.0
